showroom.js

```diff
@@ -59,7 +59,7 @@ console.log(Swiftdzire);
 var scorpio = {
   name: "Scorpio",
   model: "Storm",
-  make: "2009",
+  makingyear: "2009",
```

Comment:
@abansal26 Please use camelcase for variable name, So now It would be 'makingYear'.

Reply:
Changed makingyear with makingYear as suggested. Commit id - 0a83d756905378983abe79ebeb1ed56bc854c999